dashboard/app/models/sections/omni_auth_section.rb

```diff
@@ -29,7 +29,7 @@
 class OmniAuthSection < Section
   def self.from_omniauth(code:, type:, owner_id:, students:, section_name: 'New Section')
```

Comment:
Doesn't matter, the name ends up the same either way. Unless you feel strongly that it is more readable this way?